testbed/addons/karpenter/construct.ts

```diff
@@ -0,0 +1,61 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import iam = require('@aws-cdk/aws-iam');
+
+export interface KarpenterProps {
+    cluster: eks.Cluster;
+    namespace: string;
+}
+
+export class KarpenterController extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: KarpenterProps) {
+        super(scope, id);
+
+        const sa = props.cluster.addServiceAccount('karpenter-controller-sa', {
+            name: "karpenter",
+            namespace: props.namespace
```

Comment:
Consider defaulting this to karpenter using ?? "karpenter" so the caller doesn't always need to specify this.